src/main/java/seedu/address/logic/commands/EditAccommodationCommand.java

```diff
@@ -0,0 +1,199 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_ADDRESS;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_NAME;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_PHONE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TAG;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_ACCOMMODATIONS;
+
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.commons.util.CollectionUtil;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.accommodation.Accommodation;
+import seedu.address.model.contact.Contact;
+import seedu.address.model.contact.Phone;
+import seedu.address.model.field.Address;
+import seedu.address.model.field.Name;
+import seedu.address.model.tag.Tag;
+
+/**
+ * Edits the details of an existing accommodation in the itinerary.
+ */
+public class EditAccommodationCommand extends EditCommand {
+    public static final String SECOND_COMMAND_WORD = "accommodation";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + " " + SECOND_COMMAND_WORD
+            + ": Edits the details of the accommodation identified "
+            + "by the index number used in the displayed activity list. "
```

Comment:
Consider this change:
```suggestion
            + "by the index number used in the displayed accommodation list. "
```